server/api/routes/service.ts

```diff
@@ -5,11 +5,12 @@ export class ServiceRoutes {
   public service: ServiceController = new ServiceController();
 
   public routes(app): void {
-    app.use(StrictlyAuthorizedRoutes);
```

Comment:
@flozender is right, we need this for making sure the requests are only available when the user is logged in. Any reasons why this was removed?

Reply:
I think it was removed during a merge conflict, @anje123 can we get this back in?